src/components/TransferWidget.tsx

```diff
@@ -213,65 +202,65 @@ export const TransferWidget = ({
     try {
       setSubmitting(true)
 
-      if (
-        dAppOption === DAppOptions.LPDrain ||
-        dAppOption === DAppOptions.LPAdd
-      ) {
-        keplrHandler(walletAddress)
-        return
-      }
-
-      if (!(await checkPoolBalance())) {
-        setSubmitting(false)
-        return
-      }
-
-      const feeParam = fee.toFixed(2)
-      const params = JSON.stringify({
-        originAddress: walletAddress,
-        originChain: sourceChain,
-        targetAddress,
-        targetChain: targetChain,
-        originSymbol: sourceCurrency,
-        targetSymbol: targetCurrency,
-        amount: feeDeduct ? (+amount - fee).toFixed(8) : amount,
-        fee: feeParam,
-        htlcCreationHash: '',
-        htlcCreationVout: 0,
-        htlcExpirationTimestamp: '0',
-        htlcVersion: '',
-        senderPubKey: ''
-      })
-
-      console.log(params)
-      await fetchWrapper.post(`${backendUrl}/auth`, params)
-      const result: any = await fetchWrapper.post(
-        `${backendUrl}/submit`,
-        params
-      )
-
-      console.log(result)
-
-      if (result?.code !== 0) {
-        errorHandler(result)
-        toast.error('Failed to submit transaction!', { icon: <ErrorIcon /> })
-        setSubmitting(false)
-        return
-      }
+      // if (
+      //   dAppOption === DAppOptions.LPDrain ||
+      //   dAppOption === DAppOptions.LPAdd
+      // ) {
+      //   keplrHandler(walletAddress)
+      //   return
+      // }
+
+      // if (!(await checkPoolBalance())) {
+      //   setSubmitting(false)
+      //   return
+      // }
+
+      // const feeParam = fee.toFixed(2)
+      // const params = JSON.stringify({
+      //   originAddress: walletAddress,
+      //   originChain: sourceChain,
+      //   targetAddress,
+      //   targetChain: targetChain,
+      //   originSymbol: sourceCurrency,
+      //   targetSymbol: targetCurrency,
+      //   amount: feeDeduct ? (+amount - fee).toFixed(8) : amount,
+      //   fee: feeParam,
+      //   htlcCreationHash: '',
+      //   htlcCreationVout: 0,
+      //   htlcExpirationTimestamp: '0',
+      //   htlcVersion: '',
+      //   senderPubKey: ''
+      // })
+
+      // console.log(params)
+      // await fetchWrapper.post(`${backendUrl}/auth`, params)
+      // const result: any = await fetchWrapper.post(
+      //   `${backendUrl}/submit`,
+      //   params
+      // )
```

Comment:
We don't need to call `/auth` anymore but the call to `/submit` is still required, it's what initiates the Kima transaction.